Adyen.xcodeproj/project.pbxproj

```diff
@@ -5886,7 +5892,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = 4.7.3;
+				MARKETING_VERSION = "";
```

Comment:
Please make sure that "Update version number" action job works fine after these changes.

Reply:
Yes 👌 this `MARKETING_VERSION` is actually a bit misleading, see https://developer.apple.com/documentation/xcode/build-settings-reference#Marketing-Version